home/modules/contribute/pages/send-pr.adoc

```diff
@@ -25,14 +25,101 @@ After you complete your changes and fixed any warnings and errors reported by An
 
  $ git push origin DOC-$$$$
 
+== Send Changes for Review
+
+To send your changes for review you can:
+
+* <<pr, Submit a GitHub pull request>>, or
+* <<submit-a-gerrit-change-request,Submit a Gerrit change request>>
+
 [#pr]
-== Submit a Pull Request
+=== Submit a GitHub Pull Request
 
 After you push your working branch to your remote repository on GitHub, you can submit a pull request to the upstream repository for review.
 
 . Go to your repository on GitHub, and click btn:[Compare & pull request].
 . On the Open a Pull Request screen, update the PR title if it needs clarification, and then click btn:[Create Pull Request].
 
+=== Submit a Gerrit Change Request
+
+Some projects use the Gerrit code review process.
+The following sections describe the steps to initialize a repository with Gerrit and how to submit a change request.
+
+==== Initial Account Setup
+
+To submit your first documentation contribution you must sign the contributor license agreement.
```

Comment:
Reword to something like, "Before you submit your first contribution, you must sign..."